interface/forms/questionnaire_assessments/report.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+require_once(dirname(__FILE__) . '/../../globals.php');
```

Comment:
drop above line. not needed (I realized this when it was causing issues on the form to portal work i did in past). at some point gonna remove this line for all the forms.

Reply:
yep prob not needed.